build.gradle

```diff
@@ -42,7 +42,7 @@ dependencies {
 }
 
 group 'org.labkey.build'
-project.version = "1.44.0-SNAPSHOT"
+project.version = "1.43.2-embeddedAccessLog-SNAPSHOT"
```

Comment:
When you merge, please update to 1.45.0-SNAPSHOT as these changes seem like they are worth of a minor release change.

Reply:
Sure. Can do.
BTW, these changes aren't super urgent if you want to include your `ignoreApiDep` changes in the release as well.